test_config.placeholder.json

```diff
@@ -0,0 +1,14 @@
+{
+  "endpoints": [
+    {
+      "endpoint-type": "PI",
```

Comment:
`"endpoint-type"` can probably be left out for this sample.

Reply:
Agreed